### PR TITLE
e2e/storage-csi: replace gcr.io/gke-release to the community registry k8s.gcr.io/sig-storage

### PR DESCRIPTION
**Pull request by cpanato**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

Change the images used in the test from `gcr.io/gke-release` to use the community registry in this case  ` k8s.gcr.io/sig-storage`

Slack 🧵 https://kubernetes.slack.com/archives/C09QZFCE5/p1611747322028100

/assign @spiffxp @msau42

**Which issue(s) this PR fixes**:

Fixes partially https://github.com/kubernetes/k8s.io/issues/1525

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: test/e2e/testing-manifests/storage-csi/gce-pd/controller_ss.yaml
Comment (cpanato) [inline test/e2e/testing-manifests/storage-csi/gce-pd/controller_ss.yaml:50]: I saw there is updated images `v0.5.1` and even `v1.1.0` https://github.com/kubernetes/k8s.io/blob/master/k8s.gcr.io/images/k8s-staging-sig-storage/images.yaml#L28-L35

should we update to use the path release or upgrade to 1.1.0?

**Review by cpanato — COMMENTED** [review]
this: https://github.com/kubernetes/kubernetes/pull/98805/files#diff-1e966d5d795fba59a8ceea256e18d4390e98c4b6fc5adeb135397ab391598174R58

looks like does not exist in the community registry here this is built? or it is easier to just copy to the new registry?

File: test/e2e/testing-manifests/storage-csi/gce-pd/controller_ss.yaml
Comment (msau42) [reply test/e2e/testing-manifests/storage-csi/gce-pd/controller_ss.yaml:50]: Yeah upgrading to 1.1.0 is fine, thanks!

**Review by msau42 — COMMENTED** [review]
/approve

File: test/e2e/testing-manifests/storage-csi/gce-pd/controller_ss.yaml
Comment (spiffxp) [reply test/e2e/testing-manifests/storage-csi/gce-pd/controller_ss.yaml:50]: /hold
To address this

**Review by spiffxp — APPROVED** [review]
Will lgtm once hold is address

File: test/e2e/testing-manifests/storage-csi/gce-pd/controller_ss.yaml
Comment (cpanato) [reply test/e2e/testing-manifests/storage-csi/gce-pd/controller_ss.yaml]: done @spiffxp PTAL

how we will deal with this image `gcr.io/gke-release/gcp-compute-persistent-disk-csi-driver:v1.0.1-gke.0` per my comment https://github.com/kubernetes/kubernetes/pull/98805#pullrequestreview-584302108

Comment (cpanato) [reply test/e2e/testing-manifests/storage-csi/gce-pd/controller_ss.yaml]: upgrading to 1.1.0 looks like now we have some timeouts :/

**Review by spiffxp — COMMENTED** [review]
/retest
/lgtm
/hold cancel

**Pull request merged**
